adsp.basic/resources/language/English/strings.po

```diff
@@ -140,7 +140,7 @@ msgid "Top center"
 msgstr ""
 
 msgctxt "#30060"
-msgid "Top Back center"
+msgid "Top back center"
```

Comment:
@MartijnKaijser its hard to tell since there's no message where these labels are used and for what.

Looking at other string Im confused, and this is so buggy that using it is near enough impossible to gain some insight that way.